common/utils/category_resolver.py

```
@@ -0,0 +1,43 @@
+            A string representing the resolved insult category.
+
+        Raises:
+            ValueError: If the provided category_selection is not a valid Insult Category.
+        """
+        category_selection = category_selection.lower()
+        insult_category_dict = Resolver._generate_insult_category_dict()
+        reverse_insult_category_dict = {v: k for k, v in insult_category_dict.items()}
+
+        if category_selection in insult_category_dict:
+            return category_selection
+        elif category_selection in reverse_insult_category_dict:
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)